JsonLd/Serializer/DateTimeNormalizer.php

```diff
@@ -37,7 +38,10 @@ public function supportsNormalization($data, $format = null)
      */
     public function supportsDenormalization($data, $type, $format = null)
     {
-        return self::FORMAT === $format && 'DateTime' === $type;
+        return self::FORMAT === $format
+            && ('DateTime' === $type || '\DateTime' === $type)
+            && ((is_string($data) || (is_array($data) && isset($data['date']) && isset($data['timezone']))))
```

Comment:
I would say that we can test `isset($data['date']) && isset($data['timezone'])` in the denormalizer to be able to throw a custom exception mentioning these 2 fields.